build_tools/linux_python_package_wrapper.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
```

Comment:
Any chance we could extend linux_portable_build.py vs forking? We could add an `--action` arg that controlled what we are delegating to vs hard-coding to linux_portable_build_in_container.sh.

Reason is: we're going to need to add more controls to the image version, config options, and platform (arm vs x86 dockers), and I'd rather have all of the manylinux stuff trampoline through the same thing to keep config control manageable. I've had this stuff get out of control before...

Reply:
Sure, I was considering this but wasn't sure if it would be worth it. I'll extend `linux_portable_build.py` with what is needed.